src/lighteval/models/base_model.py

```diff
@@ -345,6 +352,97 @@ def greedy_until_with_logits(
             override_bs=override_bs,
         )
 
+    def greedy_until_multi_turn(
+        self, requests: list[GreedyUntilMultiTurnRequest], override_bs: Optional[int] = None
+    ) -> GenerateMultiTurnReturn:
+        for request in requests:
+            request.stop_sequence = as_list(request.stop_sequence) + [self.tokenizer.eos_token]
+            request.tokenized_context = self.tok_encode(request.context)
+
+        results = []
+
+        # if self.accelerator:
+        #     dataloader = self.accelerator.prepare(dataloader)
+
+        # Always batch size 1 for multi-turn
```

Comment:
If batch size is one, print a log somewhere to explain that we discard the batch override if present